### DISCUSSION TITLE
v2/zh/cookbook/tutorial/deploy

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#351] [Announcements]
# v2/zh/cookbook/tutorial/deploy

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.gitee.io/v2/zh/cookbook/tutorial/deploy.html

**Comment by Aurora-up** [comment]
大家注意部署后更改github的默认分支到你文档部署的分支，血的教训😢
